Source/Fuse.ImageTools/Tests/ImageTools.Test.uno

```diff
@@ -0,0 +1,48 @@
+using Uno;
+using Uno.Collections;
+using Uno.Testing;
+using Fuse;
+using FuseTest;
+
+namespace Fuse.ImageTools.Test
+{
+	public class ImageToolsTest : TestBase
+	{
+		private Object _image = null;
+
+		[Test]
+		[Ignore("Only supported on dotnet, Android or iOS", "native")]
```

Comment:
Yeah, right. I just looked at the current version of the implementation. YEah. `MOBILE || DOTNET` seems correct